test/suites/test-workflows/testkube-multiagent-targets-workflow.yaml

```
@@ -0,0 +1,44 @@
+apiVersion: testworkflows.testkube.io/v1
+kind: TestWorkflow
+metadata:
+  name: testkube-multiagent-targets-workflow-suite
+  labels:
+    core-tests: workflows-suite
+    type: suite
+    tool: testkube-multi-agent
+description: Run only at .dev where these runners are created!
+spec:
+  steps:
+  - execute:
+      parallelism: 2
+      workflows:
+      - name: k6-workflow-smoke
+        description: "runner: default runner(s) (no target - superagent or runner1)"
+      - name: maven-workflow-smoke-jdk11
+        description: "runner: superagent"
+        target: { match: { runnertype: [superagent] } }
```

not important but maybe more readable:
```suggestion
        target:
          match:
            runnertype:
            - superagent
```
or:

```suggestion
        target:
          match:
            runnertype: [superagent]
```

---

I used different ones on purpose, but this one-liner isn't too readable - replaced with `runnertype: [superagent]`